proto/protocolapis/astria/protocol/genesis/v1alpha1/types.proto

```diff
@@ -40,12 +41,22 @@ message IbcParameters {
   bool outbound_ics20_transfers_enabled = 3;
 }
 
-message Fees {
-  astria.primitive.v1.Uint128 transfer_base_fee = 1;
-  astria.primitive.v1.Uint128 sequence_base_fee = 2;
-  astria.primitive.v1.Uint128 sequence_byte_cost_multiplier = 3;
-  astria.primitive.v1.Uint128 init_bridge_account_base_fee = 4;
-  astria.primitive.v1.Uint128 bridge_lock_byte_cost_multiplier = 5;
-  astria.primitive.v1.Uint128 bridge_sudo_change_fee = 6;
-  astria.primitive.v1.Uint128 ics20_withdrawal_base_fee = 7;
+message GenesisFees {
```

Comment:
Personally, I feel that it makes the most sense to keep `GenesisFees` within the genesis proto, just since these fees should be strictly associated with the genesis state, but I'd be happy to move if you feel strongly about it.